skins/Seasons/skin.conf

```diff
@@ -213,6 +213,7 @@
 
     [[day_images]]
         x_label_format = %H:%M
+        x_label_spacing = 1
```

Comment:
could you include some screen shots that show the effect of these pixel-tweaking changes?  we need to be sure they do not break other configurations.